### PR TITLE
Feature/hrt358 generate csv

### PR DESCRIPTION
**Pull request by jasattilia**
## Link to JIRA ticket

Add a link to the JIRA ticket that the changes in this PR describe.

### *What is the problem we're trying to solve*

Add a summary of the problem and any additional context that helps explain what the issue we're trying to solve is.

### *What changes have we introduced*

Add descriptions of what changes were made to address the problem and provide a solution.

Include any links to commits that introduce significant additions of code if they help explain the process of coming to the solution. e.g Addition of test database setup, addition of first test and production class, removal of buggy code, etc.

#### _Checklist_

- [ ] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [ ] Added tests to cover all new production code
- [ ] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [ ] Checked all code for possible refactoring
- [ ] Code pipeline builds correctly

### *Follow up actions after merging PR*

Are there any next steps that need to addressed after merging this PR? Add them here.

### CODE REVIEW

File: HousingRegisterApi/Startup.cs [hunk 1]
```diff
@@ -102,7 +102,7 @@ public void ConfigureServices(IServiceCollection services)
                 //Get every ApiVersion attribute specified and create swagger docs for them
                 foreach (var apiVersion in _apiVersions)
                 {
-                    var version = $"v{apiVersion.ApiVersion.ToString()}";
```
Comment (jasattilia) [inline HousingRegisterApi/Startup.cs:105]: compiler advisory

File: HousingRegisterApi/V1/Functions/NovaletExportGeneratorFunction.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        /// </summary>
+        public NovaletExportGeneratorFunction()
+        {
+
+        }
+
+        public void Handle(ILambdaContext context)
+        {
+            context.Logger.LogLine("Generating Novalet export for approval");
+            var getExportFileUseCase = ServiceProvider.GetService<IGetNovaletExportUseCase>();
```
Comment (tcmorris) [inline HousingRegisterApi/V1/Functions/NovaletExportGeneratorFunction.cs:26]: Can these be injected?

File: HousingRegisterApi/V1/Infrastructure/SssInitilisationExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace HousingRegisterApi.V1.Infrastructure
+{
+    /// <summary>
+    /// Initialise the Simple Storage Service 
+    /// </summary>
+    [ExcludeFromCodeCoverage]
+    public static class SssInitilisationExtensions
```
Comment (tcmorris) [inline HousingRegisterApi/V1/Infrastructure/SssInitilisationExtensions.cs:13]: Rename to `S3InitilisationExtensions`?

**Review by tcmorris — APPROVED** [review]

File: HousingRegisterApi/V1/Functions/NovaletExportGeneratorFunction.cs (hunk 2 above)
Comment (jasattilia) [reply HousingRegisterApi/V1/Functions/NovaletExportGeneratorFunction.cs:26]: don't think so as this is the entry point

**Pull request merged**